web/src/lib/components/Navbar.svelte

```diff
@@ -72,64 +110,99 @@
     onDestroy(unsubscribe);
 </script>
 
-<div class="navbar bg-base-300">
-    <div>
-        <a href={getBaseUrl()}>
-            <img src="/images/logo.jpg" class="mr-3 h-6 sm:h-9 rounded" alt="Plebeian Technology" />
+<nav class="navbar flex items-center bg-base-300 flex-wrap">
+    <a href={getBaseUrl()} class="flex" on:click={closeMenu}>
+        <img src="/images/logo.jpg" class="w-10 h-10 rounded align-middle" alt="Plebeian Technology" />
+        <span class="md:hidden inline-block card-title text-2xl items-center text-primary ml-2">
+            Plebeian Market
+        </span>
+        {#if isLocal() || isStaging() }
+            <!-- hiddin on small devices -->
+            <div class="hidden md:inline badge badge-primary ml-2">{getEnvironmentInfo()}</div>
```

Comment:
uh, this is tricky, so on small devices we will not know that we are on staging/local? this kinda defeats the purpose of the badge, because it makes us not trust the badge anymore. honestly, i don't care if on testing/staging it looks suboptimal, but i care that i can trust the badge. otherwise just remove the badge completely.

Reply:
alternatively, use (T) / (S) in the badge maybe? as a compromise. but really, if the badge is not going to be consistent, better just kill it ... since we will end up not trusting it.